superset/daos/base.py

```diff
@@ -62,7 +62,7 @@ def find_by_id(
         model_id: str | int,
         session: Session = None,
         skip_base_filter: bool = False,
-    ) -> Model | None:
+    ) -> T | None:
```

Comment:
Can you force `T` to extend `Model` (if it already is that way, nvm)?

Reply:
it is https://github.com/apache/superset/blob/26e5a69347b1396ec7c36b7e6d2bb8d14adeb2c9/superset/daos/base.py#L35